vendor/engines/bulk_email/app/views/bulk_email/bulk_email/_export.html.haml

```diff
@@ -1,10 +1,12 @@
-= form_tag facility_bulk_email_path, :method => :get, :id => "bulk_email_export" do
+= form_tag facility_bulk_email_path, method: :get, id: "bulk_email_export" do
   = hidden_field_tag :format, :csv
-  = hidden_field_tag :search_type, @search_fields[:search_type]
+  - @search_fields[:bulk_email][:user_types].each do |user_type|
```

Comment:
Thanks - it turns out the `label` tag having the `checkbox` class is crucial. There are other checkbox labels in the app missing this; `/facilities/:id/facility_accounts/:id/edit` has one example.